x/datapool/types/pool.go

```diff
@@ -0,0 +1,72 @@
+package types
+
+import (
+	"fmt"
+	"strings"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	sdkerrors "github.com/cosmos/cosmos-sdk/types/errors"
+	"github.com/medibloc/panacea-core/v2/x/datadeal/v044_temp/address"
+)
+
+const (
+	PENDING = "PENDING"
+	ACTIVE  = "ACTIVE"
+)
+
+func NewPool(poolId uint64, curator sdk.AccAddress, poolParams PoolParams) *Pool {
+	poolAddress := NewPoolAddress(poolId)
+
+	return &Pool{
+		PoolId:       poolId,
+		PoolAddress:  poolAddress.String(),
+		Round:        1,
+		PoolParams:   &poolParams,
+		CurNumData:   0,
+		NumIssuedNft: 0,
+		Curator:      curator.String(),
+		Status:       PENDING,
+	}
+}
+
+func NewPoolAddress(poolId uint64) sdk.AccAddress {
```

Comment:
```suggestion
func newPoolAddress(poolId uint64) sdk.AccAddress {
```
pls make this function private if it's not used outside of this package.

Reply:
Also, `VerifyPoolAddressFormat` below.

Reply:
modified in [c2f5cf9](https://github.com/medibloc/panacea-core/pull/287/commits/c2f5cf906992109ff1cfd74749ebe2b0e6782b70)